pkg/command/service/clean.go

```diff
@@ -127,6 +139,14 @@ kperf service clean --namespace-prefix testns / --namespace nsname
 			} else {
 				fmt.Println("No service found for cleaning")
 			}
+
+			if cmd.Flags().Changed("verbose") {
+				fmt.Printf("[Verbose] Knative services cleaning finished.\n")
+				fmt.Printf("[Verbose] Cleaned namespaces number: %d\n", len(matchedNsNameList))
```

Comment:
The message here is confusing, in facts, namespace are not cleaned, I think we cna remove this line